dojo/settings/settings.dist.py

```diff
@@ -1260,6 +1260,7 @@ def saml2_attrib_map_format(dict):
     'KubeHunter Scan': ['title', 'description'],
     'kube-bench Scan': ['title', 'vuln_id_from_tool', 'description'],
     'Threagile risks report': ['title', 'cwe', "severity"],
+    'Humble Json Importer': ['title', 'description'],
```

Comment:
```suggestion
    'Humble Json Importer': ['title'],
```
I would like to ask, why is `'description'` needed for deduplication?